lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/test/java/io/lighty/applications/rnc/module/SwaggerTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022 PANTHEON.tech s.r.o. All Rights Reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+package io.lighty.applications.rnc.module;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import io.lighty.applications.rnc.module.config.RncLightyModuleConfigUtils;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class SwaggerTest {
+
+    private static final int MODULE_TIMEOUT = 60;
+    private static final String BASE_URL = "http://localhost:8888";
+    private static final String PRIMARY_NAME = "urls.primaryName";
+    private static final String APIDOC_INDEX = "/apidoc/explorer/index.html";
+    private static final List<String> SWAGGER_URIS = Arrays.asList(
```

Comment:
I would prefer to use: https://howtodoinjava.com/testng/testng-parameters/ instead of iterating over list.

Reply:
Done